lib/node_modules/@stdlib/math/base/special/ceil/package.json

```diff
@@ -63,5 +63,82 @@
     "nearest",
     "value",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "ceil",
+      "alias": "ceil",
+      "pkg_desc": "round a number toward positive infinity",
+      "desc": "rounds a number toward positive infinity",
+      "short_desc": "ceil",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "double",
+            "dtype": "float64"
+          },
+          "domain": [
+            {
+              "min": "-infinity",
+              "max": "infinity"
+            }
+          ],
+          "rand": {
+            "prng": "random/base/uniform",
+            "parameters": [
+              -10,
+              10
+            ]
+          },
+          "example_values": [
+            64,
+            27,
+            0,
+            0.1,
+            -9,
+            8,
+            -1,
+            125,
```

Comment:
```suggestion
            64.5,
            27.6,
            0.9,
            0.1,
            -9.2,
            8.3,
            -1.4,
            125.01,
```

Reply:
Given that this is a rounding function, we want the values to be "interesting" and not no-ops.